src/masonite/environment/environment.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, environment=None, override=True, only=None):
         if environment:
             self._load_environment(environment, override=override)
 
-        if "pytest" in sys.modules:
```

Comment:
This code is 3 years old lol. The issue is that if pytest is imported anywhere before this load environment class is called, then this will return true. So this was loading the testing env during normal development